plot_positions.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.patches as pat
+import matplotlib.axes as ax
+import matplotlib.cm as cm
+
+
+# Match variables below to run
+domain_file = "20x20rand30.dat"
+seekers = 50
+cores = 2
+domain_x = 20
+domain_y = 20
+iterations = 200
+
+# Should not need to adjust anything below here
+domain = np.genfromtxt(domain_file, skip_header=2)
+pos_data = np.zeros(14)
+
+dom_x, dom_y = domain.shape
+
+
+for time in range(0,iterations):
+    print(time)
+    plt.figure()
+    plt.title("Time = {0}".format(time))
+    ax = plt.gca()
```

Comment:
and it's kinda confusing, since here `ax` is just a regular variable which shadows the import above.